Find a functionally equivalent code:android.media.MediaPlayer.setAudioStreamType:COMMENT
<android.media.MediaPlayer: void setAudioStreamType(int)>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Sets the audio stream type for this MediaPlayer. See {@link AudioManager}
 * for a list of stream types. Must call this method before prepare() or
 * prepareAsync() in order for the target stream type to become effective
 * thereafter.
 *
 * @param streamtype the audio stream type
 * @see android.media.AudioManager
 */

@@ -1 +1,4 @@
-
+{
+    _setAudioStreamType(streamtype);
+    mStreamType = streamtype;
+}

